build.gradle.kts

```diff
@@ -33,7 +33,7 @@ subprojects {
     }
 
     dependencies {
-        testCompile(group = "junit", name = "junit", version = "+")
+        "testCompile"(create(group = "junit", name = "junit", version = "+"))
```

Comment:
I don't think these are coming back either. It will only be accessible in sub project `*.gradle.kts` files where the plugin is fully applied, not in the `subprojects` block.